packages/shared/components/profitability/profitability.helper.ts

```diff
@@ -9,25 +9,29 @@ export enum PROFITABILITY_VARIANT {
   CHIPS = "chips"
 }
 
+export const signs = {
+  negative: "-",
+  positive: "+"
+};
+export const arrows = {
+  negative: `${String.fromCharCode(8595)} `,
+  positive: `${String.fromCharCode(8593)} `
+};
+export const noPrefix = {
+  negative: "",
+  positive: ""
+};
+
 export const composeProfitabilityPrefix = (
   prefix: PROFITABILITY_PREFIX
 ): { negative: string; positive: string } => {
```

Comment:
You can create interface for `{ negative: string; positive: string }` and use it in out of `composeProfitabilityPrefix`, type of `signs`, `arrows`, `noPrefix` and other places